site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md

```diff
@@ -0,0 +1,68 @@
+
+There are also many [open issues] you can try your skills on, because who doesn't want to be [famous](https://github.com/ManageIQ/manageiq/graphs/contributors)? And, as always, we're ready to help on [gitter].
+
+
+## Improved
+
+As a sign of things to come, [Allen](https://github.com/AllenBW) has finished moving our Service UI from Bower to npm (actually, Yarn, we're so modern!) in [#296](https://github.com/ManageIQ/manageiq-ui-service/pull/296), ManageIQ itself will follow shortly.
+
+[Martin P](https://github.com/martinpovolny) has deduplicated our `tree_select` and `accordion_select` methods in [#12320](https://github.com/ManageIQ/manageiq/pull/12320), making it easier to make bigger changes to trees, and getting rid of about 40 lines in the process. While deduplicating, he also looked into `x_show` in [#12313](https://github.com/ManageIQ/manageiq/pull/12313), and apparently, he's also [removed "some code"](https://github.com/ManageIQ/manageiq/pull/12314), that's it :).
+
+[Zahi](https://github.com/zakiva) has added advanced search capability to Containers in [#12303](https://github.com/ManageIQ/manageiq/pull/12303), bringing Containers a bit closer to Infra & Cloud, feature-wise.
+
+
+## Fixed
+
+[Eric H](https://github.com/hayesr) has fixed a *weird* bug, where `Kernel.const_get` would return the wrong class for toolbars. See for yourself in [#12451](https://github.com/ManageIQ/manageiq/pull/12451).
+
+The (Self) Service UI repo got renamed, and the URL it lives on has changed from `/self_service` to `/ui/service`. [Allen](https://github.com/AllenBW) and [Satoe](https://github.com/simaishi) have been fixing the fallout, updating apache config in [#12311](https://github.com/ManageIQ/manageiq/pull/12311), and updating the Dockerfile in [#12263](https://github.com/ManageIQ/manageiq/pull/12263).
+
+
+## New
+
+[Libor](https://github.com/lpichler) has made it possible to do daily chargeback, in [#12351](https://github.com/ManageIQ/manageiq/pull/12351), following up on his earlier work fixing hourly, weekly and monthly.
+
+From the OpenStack land, [Gilles](https://github.com/gildub) has been adding support for provisioning security groups ([#11758](https://github.com/ManageIQ/manageiq/pull/11758)) and floating IPs ([#11819](https://github.com/ManageIQ/manageiq/pull/11819)), with pull requests to support these in the UI coming up shortly.
+
+In RHEV land, [Tomas](https://github.com/jelkosz) has implemented migrating VMs in [#11366](https://github.com/ManageIQ/manageiq/pull/11366).
+
+And in our API, [Alon](https://github.com/alongoldboim) has added a `MiqPolicy` collection making it possible to CRUD policies ([#11505](https://github.com/ManageIQ/manageiq/pull/11505)), and [Brandon](https://github.com/bdunne) has made it possible to approve or deny requests via the API ([#12258](https://github.com/ManageIQ/manageiq/pull/12258)).
+
```

Comment:
yeah, I started doing without, then ran out of ways to say "P has done X in pr N"

But you're right, consistency good, I'll fix it right away :)